commands/initLoggerCommands.go

```diff
@@ -0,0 +1,60 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/ditrit/verdeter"
+	verdetermodels "github.com/ditrit/verdeter/models"
+	"github.com/noirbizarre/gonja"
+)
+
+func initLoggerCommands(cfg *verdeter.VerdeterCommand) {
+
+	loggerModeKey := "logger.mode"
+	cfg.GKey(loggerModeKey, verdeter.IsStr, "", "[LOGGER] either \"prod\" or \"dev\"")
+	cfg.SetDefault(loggerModeKey, "prod")
+	cfg.SetRequired(loggerModeKey)
+	loggerModeValidator := verdetermodels.Validator{
+		Name: "Logger Mode Validator",
+		Func: func(input interface{}) error {
+			allowedModes := []string{"prod", "dev"}
+			inputStr, ok := input.(string)
+			if !ok {
+				return fmt.Errorf("logger.mode value should be a string")
+			}
+			if inputStr == "" {
+				return fmt.Errorf("logger.mode value can't be empty")
+			}
+			for _, mode := range allowedModes {
+				if mode == inputStr {
+					return nil
+				}
+			}
+			return fmt.Errorf("logger.mode should be one of the implemented modes (%v)", allowedModes)
+
+		},
+	}
+	cfg.SetValidator(loggerModeKey, loggerModeValidator)
+
+	loggerRequestTemplateKey := "logger.request.template"
+	cfg.GKey(loggerRequestTemplateKey, verdeter.IsStr, "", "[LOGGER] the request logger template")
```

Comment:
`[LOGGER] the request logger template` -> `Template message for all request logs`